### PR TITLE
Renamed the files on a branch

### PR DESCRIPTION
**Pull request by chortlehoort**
so that this code could be reviewed on a PR

### CODE REVIEW

File: quiz.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    margin-left: auto;
+    margin-right: auto;
+}
+
+footer {
+  font-size: 0.75em;
+  margin-top: 1%;
+}
+
+.second span, .third span {
```
Comment (chortlehoort) [inline quiz.css]: `second` and `third` could be named more appropriately. What if their order in the HTML changed?  When adding a class to an HTML element, think about what purpose that element is serving and give it a matching class.

For example, both of these sections could have a class named `article-section-bold`

File: quiz.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          </header>
+          <span>Size: 4" to 7" tall, 3" pot</span>
+          <span>Weight: 11oz</span>
+          <footer>Specifications valid until <time>01/31/2016</time></footer>
+        </section>
+        <section class="third">
+          <header>
+            <h2>Product Pricing</h2>
+          </header>
+          <span>1-5: $4.29</span>
```
Comment (chortlehoort) [inline quiz.html]: Instead of adding an inline element and using CSS to style it as a block element, just use a block element such as a `div` or `p` tag.

**Comment by chortlehoort** [comment]
@kdylanthomas Very good comprehension of CSS and HTML.  I have a couple of notes that I feel you should work on before I'll mark this as complete.  If you have any questions, come and snag one of us.

**Comment by kdylanthomas** [comment]
:+1:

**Pull request merged**